.github/workflows/integration_tests.yml

```diff
@@ -408,6 +408,9 @@ jobs:
           curl -vkL --silent --user ${GITEA_ADMIN_USER}:${GITEA_ADMIN_PASSWORD} -X POST "http://localhost:3000/api/v1/user/keys" -H "accept: application/json" -H "Content-Type: application/json; charset=utf-8" -d "{ \"key\": \"$GITEA_PUBLIC_KEY\",  \"title\": \"public-key-gitea\"}"
           GITEA_TOKEN=$(cat gitea-token.txt | jq -r .sha1)
           kubectl create secret generic gitea-access -n ${KEPTN_NAMESPACE} --from-literal=username=${GITEA_ADMIN_USER} --from-literal=password=${GITEA_TOKEN} --from-literal=private-key="${GITEA_PRIVATE_KEY}" --from-literal=private-key-pass=${GITEA_PRIVATE_KEY_PASSPHRASE}
+          helm repo add mockserver https://www.mock-server.com
+          helm upgrade --install --namespace ${KEPTN_NAMESPACE} --version 5.13.0 mockserver mockserver/mockserver
+          kubectl patch svc mockserver -p '{"spec": {"type": "LoadBalancer"}}' -n ${KEPTN_NAMESPACE}
```

Comment:
Why do you need a load balancer service for this?

Reply:
also, please put this in a separate step. it doesn't really have anything to do with installing gitea